scripts/globals/items/fancy_gilet.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 25774
+-- Fancy Gilet
+-- Dispenses Persikos Snow Cone
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+
+    return result
+end
+
+itemObject.onItemUse = function(target)
+    target:addItem(6565) -- Persikos Snow Cone
```

Comment:
This is the lua file Xaver referred to https://github.com/LandSandBoat/server/blob/base/scripts/globals/items.lua

I'm sure we have _many_ examples that don't do this yet, but we've started placing item IDs here and then referring to items by name elsewhere in the other Lua scripts. We use the long form name shown in the item sql where possible with a few tweaks like all caps and `_M1` for `-1` and `_p1` for `+1` items so in this case: 

`PERSIKOS_SNOW_CONE                    = 6565,`

Would be what you'd add, right after DECANTER_ARCIELA_II

Reply:
Would it be alright if I went through some of the items found in "scripts\globals\items" and update the magic numbers to ID lookups? I can make a new PR updating them

Reply:
not only alright, that would be very excellent of you! <3